hs_core/hydroshare/utils.py

```diff
@@ -442,6 +445,34 @@ def copy_resource_files_and_AVUs(src_res_id, dest_res_id):
         tgt_res.resources.set(src_res.resources.all())
 
 
+@sync_to_async
+def get_relations():
+    # TODO: not really sure why this isn't working
```

Comment:
@sblack-usu I am trying to revise these asynchronous calls so that they all use one `aiohttp.ClientSession` (instead of creating a new session for every request). However I am getting 
```
django.core.exceptions.SynchronousOnlyOperation: You cannot call this from an async context - use a thread or sync_to_async.
```
This is really my first foray into aiohttp so I'm not sure what I'm doing wrong. Would appreciate any pointers you have, thanks!

Reply:
NVM, sorry. I was able to figure it out. It was because the Django queryset is lazy, so it doesn't behave well in the async context.